scripts/sk_collect_dumps/remote.yaml

```diff
@@ -1,18 +1,25 @@
 - name: Fetch state dumps from safekeepers
-  hosts: safekeepers
+  hosts: all
   gather_facts: False
-  remote_user: "{{ remote_user }}"
+  remote_user: "developer"
     
   tasks:
-    - name: Download file
+    - name: Save JSON file to a remote disk
       get_url:
         url: "http://{{ inventory_hostname }}:7676/v1/debug_dump?dump_all=true&dump_disk_content=false"
-        dest: "/tmp/{{ inventory_hostname }}.json"
-
-    - name: Fetch file from remote hosts
-      fetch:
-        src: "/tmp/{{ inventory_hostname }}.json"
-        dest: "./result/{{ inventory_hostname }}.json"
-        flat: yes
-        fail_on_missing: no
+        dest: "/tmp/{{ inventory_hostname }}-dev.json"
 
+    - local_action: 
+        module: shell
+        cmd: |
+          set -e
+          tsh ssh "developer@{{ inventory_hostname }}" "while true; do sleep 30; done" &
```

Comment:
What is that? tsh scp doesn't work without open session?

Reply:
AFAIR, it really doesn't work without an open session

Reply:
Realised that it actually works, only very slow on prod data due to huge size. I'll try to push switch to rsync soon.